opentelemetry-semantic-conventions/src/opentelemetry/semconv/v1_23_1/db_metrics.py

```diff
@@ -0,0 +1,133 @@
+
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opentelemetry.metrics import (
+    Counter,
+    Histogram,
+    Meter,
+    UpDownCounter,
+    ObservableGauge,
+)
+from typing import Callable, Sequence
+
+class DbMetrics:
+
+    """
+    The time it took to create a new connection
+    """
+    @staticmethod
+    def create_db_client_connections_create_time(meter: Meter) -> Histogram:
```

Comment:
Similar to the comment about constants, since all the methods are static you can just make these free functions at the module level. Then they can be used like

```py
from opentelemetry.semconv.v1_23_1 import db_metrics

db_metrics.create_db_client_connections_create_time()
```